packages/testlab/src/skip-travis.ts

```diff
@@ -3,6 +3,47 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+/**
+ * A function defining a new test case or a test suite, e.g. `it` or `describe`.
+ */
+export type TestDefinition<ARGS extends unknown[], RETVAL> = (
+  name: string,
+  ...args: ARGS
+) => RETVAL;
+
+/**
+ * Helper function for skipping tests on Travis CI.
+ *
+ * @example
+ *
+ * ```ts
+ * skipOnTravis(it, 'does something when some condition', async () => {
+ *   // the test
+ * });
+ * ```
+ *
+ * @param verb - The function to invoke to define the test case or the test
+ * suite, e.g. `it` or `describe`.
+ * @param name - The test name (the first argument of `verb` function).
+ * @param args - Additional arguments (framework specific), typically a function
+ * implementing the test.
+ */
+export function skipOnTravis<ARGS extends unknown[], RETVAL>(
+  verb: TestDefinition<ARGS, RETVAL> & {skip: TestDefinition<ARGS, RETVAL>},
```

Comment:
Can we make `verb` default to `it`?

Reply:
I prefer not to. Without the verb, it would be difficult to understand whether we are defining a test case (`it()`) or a test suite (`describe()`) or something else (`context()`).

```ts
skipOnTravis('returns some value when some conditions', async () => {
 // the test
});
```